sys-apps/ignition/files/0005-config-add-ignition-translation.patch

```diff
@@ -0,0 +1,2625 @@
+From 86b42fab8a1bf218babc26176d50c54fb068bd10 Mon Sep 17 00:00:00 2001
+From: Mathieu Tortuyaux <mtortuyaux@microsoft.com>
+Date: Mon, 26 Sep 2022 12:12:04 +0200
+Subject: [PATCH 05/16] config: add ignition translation
+
+it's a merge from flatcar/ign-converter
```

Comment:
Merge in terms of copying the files unmodified? I guess it would be good to document how to redo this in the future.

Reply:
Ok, I'll add a README in the `files` section.